V 0.0.4
T 2024-03-13T11:58:55.570+00:00
ID 1|11582398-e131-11ee-ae52-0e4daf9a7381
I "sys.platform=darwin"
I "python=3.10.13 (main, Feb 12 2024, 18:19:15) [Clang 15.0.0 (clang-1500.1.0.2.5)]"
M a:"pytest"
SR a|0.002
M c:"config and collect"
M d:""
P b:c|d|d|d|0
ST b|0.002
M e:"stdout"
M f:"============================= test session starts =============================="
C e|f|0.01
M g:"\n"
C e|g|0.01
M h:"platform darwin -- Python 3.10.13, pytest-7.4.2, pluggy-1.3.0"
C e|h|0.01
C e|g|0.01
M i:"rootdir: /Users/cosminpopa/Work/RoboCorp/code/java-access-bridge-wrapper"
C e|i|0.015
C e|g|0.015
M j:"plugins: robocorp-log-pytest-0.0.3, datadir-1.5.0, timeout-2.2.0, regressions-1.0.6, xdist-3.5.0"
C e|j|0.015
C e|g|0.015
M k:"collected 0 items / 1 error"
C e|k|0.11
C e|g|0.11
M l:"PASS"
ET l|d|0.11
M n:"run finished (print results)"
P m:n|d|d|d|0
ST m|0.11
C e|g|0.131
M o:"==================================== ERRORS ===================================="
C e|o|0.131
C e|g|0.132
M p:"__________________ ERROR collecting tests/test_jab_wrapper.py __________________"
C e|p|0.132
C e|g|0.132
M q:"ImportError while importing test module '/Users/cosminpopa/Work/RoboCorp/code/java-access-bridge-wrapper/tests/test_jab_wrapper.py'.\nHint: make sure your test modules/packages have valid Python names.\nTraceback:\n../../../../.pyenv/versions/3.10.13/lib/python3.10/importlib/__init__.py:126: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n../../../PYVEnvs/pyenv3.10.13.robocorp.vscode/lib/python3.10/site-packages/robocorp/log/_rewrite_importhook.py:197: in exec_module\n    exec(co, module.__dict__)\ntests/test_jab_wrapper.py:12: in <module>\n    from JABWrapper.context_tree import ContextNode, ContextTree, SearchElement\n../../../PYVEnvs/pyenv3.10.13.robocorp.vscode/lib/python3.10/site-packages/robocorp/log/_rewrite_importhook.py:197: in exec_module\n    exec(co, module.__dict__)\nsrc/JABWrapper/context_tree.py:8: in <module>\n    from JABWrapper.jab_wrapper import JavaAccessBridgeWrapper\n../../../PYVEnvs/pyenv3.10.13.robocorp.vscode/lib/python3.10/site-packages/robocorp/log/_rewrite_importhook.py:197: in exec_module\n    exec(co, module.__dict__)\nsrc/JABWrapper/jab_wrapper.py:5: in <module>\n    from ctypes import (\nE   ImportError: cannot import name 'WINFUNCTYPE' from 'ctypes' (/Users/cosminpopa/.pyenv/versions/3.10.13/lib/python3.10/ctypes/__init__.py)"
C e|q|0.132
C e|g|0.132
M r:"=========================== short test summary info ============================"
C e|r|0.132
C e|g|0.132
M s:"ERROR tests/test_jab_wrapper.py"
C e|s|0.132
C e|g|0.132
M t:"!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!"
C e|t|0.132
C e|g|0.132
M u:"=============================== 1 error in 0.12s ==============================="
C e|u|0.132
C e|g|0.132
ET l|d|0.132
M v:"FAIL"
ER v|0.132
